exporter/signalfxexporter/translation/translator.go

```diff
@@ -68,6 +138,37 @@ func validateTranslationRules(rules []Rule) error {
 				return fmt.Errorf("only one %q translation rule can be specified", tr.Action)
 			}
 			renameDimentionKeysFound = true
+		case ActionRenameMetrics:
+			if tr.Mapping == nil {
+				return fmt.Errorf("field \"mapping\" is required for %q translation rule", tr.Action)
+			}
+		case ActionMultiplyInt, ActionDivideInt:
+			if tr.ScaleFactorsInt == nil {
```

Comment:
It would be worth to check that the divisor is not 0.

Reply:
added the check on zero